_includes/components/gc-features/features.html

```diff
@@ -0,0 +1,31 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of features
+++++++++++++++++++++++++++++++++++
+
```

Comment:
This file seem to be a duplicate of "components/gc-features/samples"

Reply:
It is, the `grunt` task copies the file. I can remove it, but anyone running `grunt` would have it copied.